src/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+   <link
+			rel="stylesheet"
+			href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+			integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm"
+			crossorigin="anonymous"
+		/>
+    <link rel="stylesheet" href="path/to/font-awesome/css/font-awesome.min.css">
+  <title>Document</title>
+</head>
+  <body>
+    <main class="wrapper">
+    <header class="list-title">
+      <h1>Todo list Items</h1>
+      <p><i class="fas fa-sync-alt "></i></p>
+    </header>
+  <div class="main-panel">
+    <input id="add-input" class="add-input" type="text" placeholder="Add to your list">
+    <ul id="todo-list">
+
+    </ul>
+    <button id="clear-btn" class="clear-btn" type="button" name="button">Clear all completed</button>
+  </div>
+</main>
+<script type ="module" src="index.js"></script>
```

Comment:
A good practice to follow is that in the template file we declare in webpack, we should not include any link tag or script tag. My suggestion is to include them through webpack process.